tests/console/wget_https.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case 1461937 - FIPS: wget
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+
+    select_console "root-console";
+    assert_script_run("rpm -q wget");
+    assert_script_run("wget -c https://build.opensuse.org -O opensuse.html");
+    wait_still_screen;
+    assert_script_run("wget -c https://www.google.com -O google.html");
+    wait_still_screen;
+    assert_script_run("wget -c https://github.com -O github.html");
+    wait_still_screen;
+    assert_script_run("ll | grep -E 'opensuse.html|google.html|github.html'");
+    assert_script_run("rm -f opensuse.html google.html github.html");
```

Comment:
I realize the API of **assert_script_run** can not  check the script result whether can be passed. So I talked with Wei Jiang about it. and he suggest me to use the sub of **zypper_call** that is in the **os-autoinst-distri-opensuse/lib/utils.pm**, and I write a sub function called **test_call** based on the **zypper_call**. to check the right exit code. and it will be in my test case.